challenge-13/submissions/onomica/solution-template.go

```
@@ -0,0 +1,174 @@
+		query.WriteString(" WHERE category = ?")
+	}
+
+	rows, err := ps.db.Query(query.String(), category)
+	if err != nil {
+		return nil, err
+	}
+
+	var products []*Product
+	p := Product{}
+
+	defer rows.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `rows.Close` is not checked

(errcheck)